build.sh

```diff
@@ -25,12 +25,16 @@ case $(uname -s) in
         werror=""
 esac
 
+io_uring=""
+brickmux=""
 case $JOB_NAME in
     'gh_regression-test-with-multiplex'|'gh_regression-on-demand-multiplex')
         brickmux="--enable-brickmux"
         ;;
+    'gh_smoke-centos7'|'gh_devrpm-el7')
+        io_uring="--disable-linux-io_uring"
+        ;;
     *)
```

Comment:
io_uring=""
brickmux=""
can be passed here. Or you can just totally remove the *) condition for the rest of the jobs.

Reply:
Removed the *). We need to handle all options in all cases, so having it on the top is better..